server/graphql/reports/src/mutations/install.rs

```diff
@@ -0,0 +1,30 @@
+use async_graphql::Context;
+use async_graphql::*;
+use graphql_core::{
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    UploadedFile,
+};
+use util::format_error;
+
+pub fn install_uploaded_reports(ctx: &Context<'_>, file_id: String) -> Result<Vec<String>> {
```

Comment:
Was just reflecting about the name, kept thinking this whole time that it should just be `upload`. Install makes me think that I'm about to install something to my machine. Though I get why we name it this way